cloud/aws/lambda/variables.tf

```diff
@@ -0,0 +1,359 @@
+  description = "Critical threshold for errors detector"
+  type        = number
+  default     = 3
+}
+
+variable "errors_threshold_warning" {
+  description = "Warning threshold for errors detector"
+  type        = number
+  default     = 1
+}
+
+variable "errors_aperiodic_duration" {
+  description = "Duration for the errors block"
+  type        = string
+  default     = "10m"
+}
+
+variable "errors_aperiodic_percentage" {
+  description = "Percentage for the errors block"
+  type        = number
+  default     = 0.9
+}
+
+# Throttles detectors
+
+variable "throttles_disabled" {
+  description = "Disable all alerting rules for throttles detector"
+  type        = bool
+  default     = null
+}
+
+variable "throttles_disabled_critical" {
+  description = "Disable critical alerting rule for throttles detector"
+  type        = bool
+  default     = null
+}
+
+variable "throttles_disabled_warning" {
+  description = "Disable warning alerting rule for throttles detector"
+  type        = bool
+  default     = null
+}
+
+variable "throttles_notifications" {
+  description = "Notification recipients list for every alerting rules of throttles detector"
+  type        = list
+  default     = []
+}
+
+variable "throttles_notifications_warning" {
+  description = "Notification recipients list for warning alerting rule of throttles detector"
+  type        = list
+  default     = []
+}
+
+variable "throttles_notifications_critical" {
+  description = "Notification recipients list for critical alerting rule of throttles detector"
+  type        = list
+  default     = []
+}
+
+variable "throttles_aggregation_function" {
+  description = "Aggregation function and group by for throttles detector (i.e. \".mean(by=['host'])\")"
+  type        = string
+  default     = ".sum(by=['FunctionName'])"
+}
+
+variable "throttles_transformation_function" {
+  description = "Transformation function for throttles detector (mean, min, max)"
+  type        = string
+  default     = "sum"
+}
+
+variable "throttles_transformation_window" {
+  description = "Transformation window for throttles detector (i.e. 5m, 20m, 1h, 1d)"
+  type        = string
+  default     = "1h"
+}
+
+variable "throttles_threshold_critical" {
+  description = "Critical threshold for throttles detector"
+  type        = number
+  default     = 3
+}
+
+variable "throttles_threshold_warning" {
+  description = "Warning threshold for throttles detector"
+  type        = number
+  default     = 1
+}
+
+variable "throttles_aperiodic_duration" {
+  description = "Duration for the throttles block"
+  type        = string
+  default     = "10m"
+}
+
+variable "throttles_aperiodic_percentage" {
+  description = "Percentage for the throttles block"
+  type        = number
+  default     = 0.9
+}
+
+# invocations detectors
+
+variable "invocations_disabled" {
+  description = "Disable all alerting rules for invocations detector"
+  type        = bool
+  default     = null
```

Comment:
should be true

Reply:
you want the detector disabled by default?

Reply:
We have been setting detectors_disabled to false in all modules and the the disabled variables for specific detectors to null

Reply:
see my answer for sqs

Reply:
done